### DISCUSSION TITLE
newbie in laravel

### DISCUSSION THREAD

**Discussion by ioshRiu** [Attendize/Attendize#1131] [Show and tell]
can someone show it to me how to use it because i got error said about autoload error so can someone help and show it to me how to

**Comment by flavius-constantin** [comment]
Can you post the error?

**Reply by ioshRiu** [reply]
![image](https://github.com/Attendize/Attendize/assets/163767746/24c4a7e9-e6af-45a9-b760-e2f74132cda2)
here

**Reply by flavius-constantin** [reply]
Did you ran **composer** **install**?

**Reply by ioshRiu** [reply]
yes i already did
but this came 
![image](https://github.com/Attendize/Attendize/assets/163767746/c398ed1a-a1b6-42ca-8cd0-29479127c234)

**Reply by ioshRiu** [reply]
im sorry im still new to laravel so i didn't know much

**Reply by gustavorieg** [reply]
install composer first

https://getcomposer.org/

**Reply by ioshRiu** [reply]
already done everything but maybe i miss something cuz i just want to localhost it

**Reply by flavius-constantin** [reply]
I suggest to use Herd: https://herd.laravel.com/windows

Try to get ready with Herd. ;-)
It is a gamechanger!
